frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
+- Prepare your dataset and email our curators for further assistance in the process
+- Publish!
+
+
+
+## Dataset Requirements
+
```

Comment:
I suggest we add an eligibility section here, to match my two formatting suggestions above. 

```suggestion
## Dataset Requirements

### Data Eligibility

Cellxgene is focused on supporting the global community attempting to create references of human cells and tissues. As a result, there are a few kinds of datasets that we are **not** likely to accept at this time:
- drug screens
- cell lines
- organisms other than mouse or human 

### Formatting Requirements

```

@jahilton you're correct that we haven't held a hard line on any of these three criteria so we shouldn't *prohibit*, but I do want to give submitters a sense of what might be rejected so we don't set up incorrect expectations. 

For cell lines, I regret the covid infected cells collection. The explorer interface is built for identifying discrete cell types and is not great for measuring subtle changes in phenotype introduced by drug screens/perturbations. If we wanted to support those data types in the future, I think we'd seriously consider building new interfaces.

Reply:
add in my new amendment to the commit

Reply:
The softer language makes sense to me, as is the need to steer away from these data types.